index.html

```diff
@@ -54,23 +52,26 @@ <h2 style="margin-bottom:0px;">RoHOW</h2>
             <img src="{{site.path}}/_photos/rohow.JPG" />
         </figure><div class="col-6-10">
             <p>
-            Seit 2014 veranstalten wir jedes Jahr den <i>Robotic Hamburg
-            Open Workshop</i> (RoHOW).
+            Since 2014, we are anually hosting the <i>Robotic Hamburg Open
+              Workshop</i> (RoHOW).
             </p>
             <p>
-            Der RoHOW ist eine
-            Weiterbildungs- und Vernetzungsveranstaltung der Technischen
-            Universität Hamburg mit über hundert Studierenden und Wissenschaftler/innen
-            aus der ganzen Welt, die auf dem Gebiet der Forschung an
-            humanoiden Robotern tätig sind.
+            The RoHOW is an event hosted by the Hamburg University of
+            Technology that serves the purpose of education and networking.
+            Over 100 scientists and students from all over the world who are
+            researching in the field of humanoid robotics participate in this
+            event.
             </p>
             <p>
-            Das Ziel der Veranstaltung ist den Austausch zwischen den
-            Universitäten und Forschungseinrichtungen durch Fachvorträge,
-            Workshops, Wettkämpfe und das persönliche Gespräch zu fördern.
+            The goal of this event is the exchange of information between
+            RoboCup teams from different universities and research institutes.
+            Participants have the opportunity to visit lectures and workshops,
+            challenge other teams in competitions, and get in direct contact
+            with each other.
             </p>
             <p>
-            Weiterführende Informationen finden sich auf <a href="https://www.rohow.de">https://www.rohow.de</a>
+            More information on <a
+              href="https://www.rohow.de">https://www.rohow.de</a>

```

Comment:
While we are on it - this link is deprecated.

```suggestion
              href="https://rohow.de">https://rohow.de</a>
```